2020-09-05 22:10:50,524 main ERROR Unable to move file d:\Files\Minecraft\1.16.2 Server\logs\latest.log to d:\Files\Minecraft\1.16.2 Server\logs\2020-09-05-2.log: java.nio.file.FileSystemException d:\Files\Minecraft\1.16.2 Server\logs\latest.log -> d:\Files\Minecraft\1.16.2 Server\logs\2020-09-05-2.log: The process cannot access the file because it is being used by another process 
2020-09-05 22:10:50,530 main ERROR Unable to delete file d:\Files\Minecraft\1.16.2 Server\logs\latest.log: java.nio.file.FileSystemException d:\Files\Minecraft\1.16.2 Server\logs\latest.log: The process cannot access the file because it is being used by another process 
[22:10:53] [main/INFO]: Environment: authHost='https://authserver.mojang.com', accountsHost='https://api.mojang.com', sessionHost='https://sessionserver.mojang.com', name='PROD' 
[22:10:53] [main/FATAL]: Failed to start the minecraft server 
java.io.IOException: The process cannot access the file because another process has locked a portion of the file 
	at sun.nio.ch.FileDispatcherImpl.write0(Native Method) ~[?:?] 
	at sun.nio.ch.FileDispatcherImpl.write(FileDispatcherImpl.java:68) ~[?:?] 
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(IOUtil.java:113) ~[?:?] 
	at sun.nio.ch.IOUtil.write(IOUtil.java:58) ~[?:?] 
	at sun.nio.ch.FileChannelImpl.write(FileChannelImpl.java:286) ~[?:?] 
	at aeu.a(SourceFile:40) ~[server.jar:?] 
	at cyd$a.<init>(SourceFile:287) ~[server.jar:?] 
	at cyd.c(SourceFile:275) ~[server.jar:?] 
	at net.minecraft.server.Main.main(SourceFile:119) [server.jar:?] 
